components/event/EventCard.js

```diff
@@ -35,6 +35,26 @@ export default function EventCard({ event, username }) {
             )}
         </div>
         <div className="flex-1 space-y-1">
+          <div className="flex space-x-2">
+            <p>
```

Comment:
We can add a condition in case the user forgot to mention the fields 
![image](https://user-images.githubusercontent.com/89926834/214070501-1bab78f2-1eed-4395-a834-7ba9d5933459.png)